apprentice_teacher.rb

```diff
@@ -1,21 +1,36 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
-
-  def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 800
-    @phase = 3
-  end
+require_relative 'teacher'
+require_relative 'praise'
 
-  def offer_high_five
-    "High five!"
-  end
+class ApprenticeTeacher < Teacher
+  
+  include Praisable
+  # Here i want to only take from the super class these attributes.
+  # But I don't know how to do that
```

Comment:
You don't pick and choose what to take from your parent class.  You get it all.  There are technically workarounds for this, but you should set it up so that only the things that all subclasses should have are in the parent class, and any children that need additional things define them themselves.